### PR TITLE
Improve Pick Place demo

### LINKED ISSUE #68: Gripper Issue
Hi there,
I wasn't able to open the gripper of pr2 with your code. I tried thatstuff but it didn't work either.(i just added the effort) Do you have any idea what should i do to open the gripper with moveit? 
Thank you in advance

```
`   g.pre_grasp_posture.joint_names.resize(1, "r_gripper_joint");
    g.pre_grasp_posture.points.resize(1);
    g.pre_grasp_posture.points[0].positions.resize(1);
    g.pre_grasp_posture.points[0].positions[0] = 1;
    g.pre_grasp_posture.points[0].effort.resize(1);
    g.pre_grasp_posture.points[0].effort[0]=1;`

```

### PR DESCRIPTION
**Pull request by v4hn**
Thanks to @ilaplace for pointing this out!

Fixes #68

### CODE REVIEW

File: pr2_moveit_tutorials/pick_place/src/pick_place_tutorial.cpp [hunk 1]
```diff
@@ -43,13 +43,52 @@
...
+  posture.joint_names[1] = "r_gripper_motor_screw_joint";
+  posture.joint_names[2] = "r_gripper_l_finger_joint";
+  posture.joint_names[3] = "r_gripper_r_finger_joint";
+  posture.joint_names[4] = "r_gripper_r_finger_tip_joint";
+  posture.joint_names[5] = "r_gripper_l_finger_tip_joint";
+
+  posture.points.resize(1);
+  posture.points[0].positions.resize(6);
+  posture.points[0].positions[0] = 0;
+  posture.points[0].positions[1] = 0;
```
Comment (dornhege) [inline pr2_moveit_tutorials/pick_place/src/pick_place_tutorial.cpp:77]: Where did you get these values from? I am using 0, 0.002, 0.002, 0.002, 0.002. I believe these were determine from an actual closing in gazebo.

Comment (v4hn) [inline pr2_moveit_tutorials/pick_place/src/pick_place_tutorial.cpp:77]: On Thu, Jul 28, 2016 at 07:39:42AM -0700, Christian Dornhege wrote:

> Where did you get these values from? I am using 0, 0.002, 0.002, 0.002, 0.002. I believe these were determine from an actual closing in gazebo.

I guessed :-)

You are short of one number. The gripper has 6 joints, not 5.
If you could give me a full list of joint-values for a closed state, I'm happy to amend the request.

Comment (dornhege) [inline pr2_moveit_tutorials/pick_place/src/pick_place_tutorial.cpp:77]: I'm not setting the r_gripper_joint for visualization, so I'd assume:
0, 0, 0.002, 0.002, 0.002, 0.002.

**Comment by dornhege** [comment]
Please rebase on #78 to make CI work.

**Comment by v4hn** [comment]
I updated the values for the closed gripper and rebased the request. Thanks!

**Pull request merged**